### ISSUE TITLE
Decide if we can make vets-api-mockdata public

### ISSUE THREAD

**Issue by gunsch** [department-of-veterans-affairs/va.gov-team#9291]
## Issue Description

`vets-api-mockdata` is one of the repositories we suggest checking out as part of the [getting started](https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/getting-started steps, but it's currently locked down.

I have yet to find any reasoning for it being private, and signs point to it just being that way historically (it's the default setting, too).

@annaswims:
> No idea on that one. My guess would be that it’s either because external services want their data private or that the VA gets weird bout ssn’s and that stuff, even if it’s fake

@kreek:
> I think it was so the shape of our data wasn’t public, but I’m not sure that it needs to be. All the data should be fake users from staging envs

@johnpaulashenfelter:
> While i don’t, i wonder if there’s some staging user data in there that there was PII worry? Can’t imagine how that would be but 🤷 

---
## Tasks
- [ ] Decide if we can make it public
- [ ] File ticket with OIT about making this repo public

## Acceptance Criteria
- [ ] vets-api-mockdata is public

**Comment by gunsch** [comment]
one more, from @patrickvinograd:
> I think there’s a fair chance it will set off some VA repo scanner out there and cause churn/confusion about OMG SSNs WTF
> other than that, I can’t think of any actual risk
> it’s possible some teams out there consider their API response formats `VA sensitive` or whatever that privacy classification is.

**Comment by omgitsbillryan** [comment]
+1 to it will set off some VA scanner and cause confusion. This already happened with the localhost SAML key & cert file that we decided to just delete rather than try to argue they weren't sensitive.

IIRC at the time it was thought, "we'd probably just better make this private to save ourselves a headache of having to explain it to someone". It was also easier back then, since so few people needed access to it.

Maybe knowing the shape of our data and where we get it from might give an attacker some additional data... eh... but the data itself is fake and not sensitive.

We'd also want to make sure to lock down a public repo more than we do now - which allows direct commits to `master`.

**Comment by gunsch** [comment]
re: "knowing the shape of our data". I was under the impression that could already be gleaned from reading vets-api. is that not true?

**Comment by wyattwalter** [comment]
Could always file a ticket with the GitHub team to ask how the scanning works and see if there's a way we can flag something as test data. Easiest path is definitely to just leave it, though.

**Comment by johnpaulashenfelter** [comment]
TL;DR from eng leads meeting today is that @gunsch is going to own, we generally want to default to get it public, and we're worried that it could trigger security flags from security (because of fake SSNs, user names, etc)

**Comment by gunsch** [comment]
immediate next steps are @wyattwalter to dig into:
* what triggered the private key incident
* see if Boris has any extra context or contacts that can help us affirm what kinds of content will cause problems

I'll talk with Dror as well and see if he's able to chase down any leads.

**Comment by gunsch** [comment]
Dror's take: "not very important, let's revisit in Q3."

**Comment by gunsch** [comment]
Whoops, meant to follow up on above. I think we should still continue the next steps --- particularly digging into what triggered the private key incident, and see what we can learn about this kind of failure mode. Once we have that, let's re-evaluate if we feel more confident or not, or decide if we should put this on the backburner for the time being.

**Comment by wyattwalter** [comment]
I went back to find the history on the incident we had called when the localhost key was discovered on vets-api. I have PDFs from it, but didn't want to upload here on the public repo since it has responder info, but basically the gist is:

- some CSOC scanner picked up that there was a private key uploaded, went to a GitHub management team to find out who owned that repo
- Angela picked it up (she might be on the GitHub team? I'm not sure)
- Key was identified as being a localhost dev key that couldn't authenticate, but it didn't matter. standard procedures were required:
  - Response team pushed on revoking the key (it was self-signed, but they didn't seem to get that or care), "is it possible for bad actors to gain access to dev environment?" (legit question, but the answer was no and it didn't matter much), when can the history of this repo be expunged and who is doing that?

The risks here are relatively high that we'd have to take some drastic steps if something was identified as having the same shape as PII or got matched by some regex scanning tool. It didn't seem to matter to them that the data exposed was fake or not, processes had to be followed.

While I would normally opt for being public, I think the risk here is fairly high that we'd get ordered to make it private again after a bunch of extra work thrown in just for being public.
